earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -101,13 +101,23 @@ void PacketGenAssetVersionImplD::UpdateChildren(
   std::vector<MutableAssetVersionD> kids;
   kids.reserve(numLevels);
 
+  std::chrono::high_resolution_clock::time_point prevTime = std::chrono::high_resolution_clock::now();
+  std::chrono::high_resolution_clock::time_point curTime;
+  std::chrono::duration<double> timeDiff; 
+  notify(NFY_PROGRESS, "UpdateChildren starting for %s", assetRef.c_str());
```

Comment:
I know I said these should be logged at the PROGRESS level, but since this is so detailed it might be better to put it at a higher level (maybe NFY_INFO2)? That way we could turn off this logging and keep the current progress logging.